### PR TITLE
[FIX] project: fix broken layout in activity view

### PR DESCRIPTION
**Pull request by yava-odoo**
In this PR, we fixed the broken layout of the task activity view.

task-3231708

### CODE REVIEW

**Review by auon-odoo — CHANGES_REQUESTED** [review]
Thank you for the fix :wrench:

File: addons/project/views/project_task_views.xml [hunk 1]
```diff
@@ -742,9 +742,9 @@
                     <field name="project_id"/>
                     <templates>
                         <div class="justify-content-between" t-name="activity-box">
-                            <div class="text-end col-6">
+                            <div class="col-6">
```
[the remark this answers is not in the document]
Comment (yava-odoo) [reply addons/project/views/project_task_views.xml]: Changes are applied thanks :slightly_smiling_face:

**Review by auon-odoo — APPROVED** [review]
Lgtm, thank you! :fire:

**Review by xavierbol — APPROVED** [review]
LGTM, thanks for your work. 🙂 

robodoo r+

**Pull request closed without merging**